lib/client/renderer.js

```diff
@@ -111,7 +111,7 @@ function init (client, d3) {
           return d.type === 'forecast' ? 'none' : d.color;
         })
         .attr('opacity', function(d) {
-          return d.noFade ? 100 : chart().futureOpacity(d.mills - client.latestSGV.mills);
+          return d.noFade || !client.latestSGV ? 100 : chart().futureOpacity(d.mills - client.latestSGV.mills);
```

Comment:
At the very first start there is no measured values and the user can see errors in console.